### ISSUE TITLE
Question: RE Integration with node.js

### ISSUE THREAD

**Issue by elliothogg** [colinbdclark/osc.js#112]
Hi there,

Would it be possible to integrate osc.js with this? 

https://github.com/Emotiv/cortex-example/tree/master/nodejs

Any help would be greatly appreciated.

Thanks

**Comment by colinbdclark** [comment]
Yes, it probably is possible to integrate osc.js with this library, but I haven't used it myself. Unfortunately, I don't know enough about your use case to give you much advice but it sounds like a really cool idea. Good luck with your project!
